README.md

```diff
@@ -7,38 +7,42 @@ Pull requests are welcomed!
 ```python
 import fsspec
 
-with fsspec.open("slk://arch/project/user/file", "r") as f:
+with fsspec.open("slk:///arch/project/user/file", "r") as f:
     print(f.read())
 ```
+### Loading datasets
+
+```python
+
+import ffspec
+import xarray as xr
+
+url = fsspec.open("slk:////arch/project/file.nc", slk_cache="/scratch/b/b12346").open()
+dset = xr.open_dataset(url)
+```
+
 
 ## Usage in combination with preffs
 ### Installation of additional requirements
-```
-git clone git@github.com:observingClouds/slkspec.git
-cd slkspec
+```console
 mamba env create
 mamba activate slkspec
-pip install .
-```
-
-### Loading dataset
-
-Set location for tape archive retrievals on local machine
-```bash
-export SLK_CACHE="/path/to/local/cache/directory/"
-```
-Load the slk module
-```bash
-module load slk
+pip install .[preffs]
 ```
 
 Open parquet referenced zarr-file
 ```python
 import xarray as xr
-ds=xr.open_zarr(f"preffs::/path/to/preffs/data.preffs", storage_options={"preffs":{"prefix":"slk:///arch/<project>/<user>/slk/archive/prefix/"}
+ds = xr.open_zarr(f"preffs::/path/to/preffs/data.preffs",
+                storage_options={"preffs":{"prefix":"slk:///arch/<project>/<user>/slk/archive/prefix/"}
 ```
-Now only those files are retrieved from tape which are needed for any requested dataset operation. In the beginning only the file containing the metadata (e.g. .zattrs, .zmetadata) and coordinates are requested (e.g. time). After the files have been retrieved once, they are saved at the path given in `SLK_CACHE` and accessed directly from there.
+
+Now only those files are retrieved from tape which are needed for any requested
+dataset operation. In the beginning only the file containing the metadata
+(e.g. .zattrs, .zmetadata) and coordinates are requested (e.g. time). After the
+files have been retrieved once, they are saved at the path given in
+`SLK_CACHE` and accessed directly from there.
+
 
 ## Current limitations (among others)
-- tape retrievals are currently not combined, leading to single/inefficient requests
```

Comment:
still true, so far ( see my one of my other comments)

Reply:
Actually, the `_retrieve_items` method gets only called once, at which point it has information on all items that need to be downloaded. So the retrievals are combined - somewhat. There are limitations though.

Reply:
We could do a `pyslk.slk_recall(SEARCH_ID_ALL_FILES)`, first, and, then, the individual retrievals.